notebooks/DAAC_UWG_cloud_tutorial/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -0,0 +1,430 @@
+    "\n",
+    "As an example data  collection, we use ICESat-2 Land Ice Height (ATL06) over the Juneau Icefield, AK, for March 2003. ICESat-2 data granules, including ATL06, are stored in HDF5 format. We demonstrate how to open an HDF5 granule and access data variables using `xarray`. Land Ice Heights are then plotted using `hvplot`. \n",
+    "\n",
+    "`earthaccess` is a package developed by Luis Lopez (NSIDC developer) to allow easy search of NASA CMR and download of NASA data collection.  It can be used for programmatic search and access for both _DAAC-hosted_ and _cloud-hosted_ data. It manages authenticating using Earthdata Login credentials which are then used to obtain the S3 tokens that are needed for S3 direct access. https://github.com/nsidc/earthaccess\n",
+    "\n",
+    "\n",
+    "### **Credits**\n",
+    "\n",
+    "The notebook was created by Andy Barrett, NSIDC, and is based on notebooks developed by Luis Lopez and Mikala Beig, NSIDC.\n",
+    "\n",
+    "For questions regarding the notebook, or to report problems, please create a new issue in the [NSIDC-Data-Tutorials repo](https://github.com/nsidc/NSIDC-Data-Tutorials/issues).\n",
+    "\n",
+    "### **Learning Objectives**\n",
+    "\n",
+    "By the end of this demonstration we will be able to:  \n",
+    "1. use `earthaccess` to search for ICESat-2 data using spatial and temporal filters and explore search results;  \n",
+    "2. open data granules using direct access to the ICESat-2 S3 bucket;  \n",
+    "3. load a HDF5 group into an `xarray.Dataset`;  \n",
+    "4. visualize the land ice heights using `hvplot`.  \n",
+    "\n",
+    "### **Prerequisites**\n",
+    "\n",
+    "1. We will be accessing data in Amazon Web Services (AWS) Simple Storage Service (S3) buckets. This requires an AWS account, for details on how to do set one up see [here](https://nsidc.org/data/user-resources/help-center/nasa-earthdata-cloud-data-access-guide#anchor-1).\n",
+    "2. **NASA cloud-hosted data is in Amazon Region us-west2. So we also need an EC2 instance in the us-west-2 region.** Details on how to set an instance up can be found [here](https://nsidc.org/data/user-resources/help-center/nasa-earthdata-cloud-data-access-guide#anchor-1).\n",
+    "3. An Earthdata Login is required for data access. If we don't have one, we can register for one [here](https://urs.earthdata.nasa.gov/).\n",
+    "4. Current recommended practice for authentication is to create a .netrc file in our home directory following [these instructions](https://nsidc.org/support/how/how-do-i-programmatically-request-data-services) (Step 1) and to use the .netrc file for authentication when required for data access during the tutorial.\n",
+    "5. The *nsidc-tutorials* environment is setup and activated. This [README](https://github.com/nsidc/NSIDC-Data-Tutorials/blob/main/README.md) has setup instructions.\n",
```

Comment:
I like how 5. this is worded, starting with a statement describing what needs to happen in order for the notebook to run. Could we change the others to match this style? So something like:
1. An Amazon Web Services (AWS) account is created. We will be accessing data...
2. An EC2 Instance in us-west-2 is setup...
3. This one is already good to go, starting with "An Earthdata Login account..."!
4. A .netrc file is created in your home directory...

Reply:
Agreed, will make these changes.